docs/developer-docs/latest/developer-resources/database-apis-reference/query-engine/single-operations.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Single Operations for Query Engine API - Strapi Developer Documentation
+description: (add description here)
+---
+<!-- TODO: update SEO tags -->
+
+# Query Engine API: Single Operations
+
+## findOne()
+
+Finds the first entry matching the parameters.
+
+Syntax: `findOne(parameters) ⇒ Entry`
+
+### Parameters
+
+| Parameter    | Type                           | Description                                |
+| ------------ | ------------------------------ | ------------------------------------------ |
+| `select`       | String, or Array of strings                     | Select some [attributes](/developer-docs/latest/development/backend-customization/models.md#model-attributes) to return           |
+| `where`       | [`WhereParam`<Fa-Link color="grey"/>](/developer-docs/latest/developer-resources/database-apis-reference/query-engine/filtering.md)     | Filter                                     |
+| `limit`        | Number                       | Always `1` when used with `findOne`        |
+| `offset`       | Number                       | How many entries to skip                   |
+| `orderBy`      | [`OrderByParam`<Fa-Link color="grey"/>](/developer-docs/latest/developer-resources/database-apis-reference/query-engine/order-pagination.md)    | How to order the entries                   |
+| `populate`     | [`PopulateParam`<Fa-Link color="grey"/>](/developer-docs/latest/developer-resources/database-apis-reference/query-engine/populating.md) | Select which relations should be populated |
+
+### Example
+
+```js
+const entry = await db.query('article').findOne({
+  select: ['title', 'description'],
+  where: { title: 'Hello World' },
+  orderBy: { title: 'DESC' },
```

Comment:
ordering by title, since the title is the same for all entries, is unnecessary. However we understand well the syntax